templates/server-acl-init-cleanup-rolebinding.yaml

```diff
@@ -2,17 +2,18 @@
 {{- if (or $serverEnabled .Values.externalServers.enabled) }}
 {{- if .Values.global.acls.manageSystemACLs }}
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
+kind: RoleBinding
 metadata:
   name: {{ template "consul.fullname" . }}-server-acl-init-cleanup
+  namespace: {{ .Release.Namespace }}
   labels:
     app: {{ template "consul.name" . }}
     chart: {{ template "consul.chart" . }}
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
 roleRef:
   apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
+  kind: Role
   name: {{ template "consul.fullname" . }}-server-acl-init-cleanup
 subjects:
   - kind: ServiceAccount
```

Comment:
I think it makes sense to remove the
```yaml
    namespace: {{ .Release.Namespace }}
```

line below for all rolebindings since the namepace is implied now.

Reply:
Ah good catch on both points!
I'll update the rolebindings in this and the other PRs related to clusterroles to remove the duplicate namespace entry from the `subject:`!
Thanks!